src/LuaTransformer.ts

```diff
@@ -147,7 +141,7 @@ export class LuaTransformer {
     public transformStatement(node: ts.Statement): StatementVisitResult {
         // Ignore declarations
```

Comment:
We no longer just 'ignore' declarations, so this comment should probably be removed.